engine/runtime/src/main/java/org/enso/interpreter/runtime/type/Types.java

```diff
@@ -221,6 +224,10 @@ private static TypeGraph buildTypeHierarchy() {
     graph.insert(ConstantsGen.PANIC, ConstantsGen.ANY);
     graph.insert(ConstantsGen.REF, ConstantsGen.ANY);
     graph.insert(ConstantsGen.TEXT, ConstantsGen.ANY);
+    graph.insert(ConstantsGen.DATE, ConstantsGen.ANY);
```

Comment:
Looks like an omission from my earlier integration, right? What gets broken when the `graph` isn't up-to-date?

Reply:
Nothing, apparently :shrug: